### PR TITLE
Add a Chinese story.

### PR DESCRIPTION
**Pull request by wkvictor**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2711)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: Chinese/Wolf/fight_wolf.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+晚上，一只大灰狼溜进了树林，踩疼了一串喇叭花。喇叭花闻出了大灰狼味，立刻变成了会报警的小喇叭：
```
Comment (casey-collab) [inline Chinese/Wolf/fight_wolf.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**